demo/site/preBuild/src/createLinkRedirectDestination.ts

```diff
@@ -0,0 +1,17 @@
+import { ExternalLinkBlockData, InternalLinkBlockData, LinkBlockData, NewsLinkBlockData } from "../../src/blocks.generated";
+
+export function createLinkRedirectDestination(content: LinkBlockData) {
```

Comment:
This function is also used in the "actual" site source code. I'd therefore move it into the src/ folder